plugin.rb

```diff
@@ -63,16 +63,18 @@ def top_posters
       data-bsa-placement="sitepointforums"></script>
     EOS
 
-  sitepoint_site_customization = SiteCustomization.find_or_create_by({
-    name: "SitePoint Crawler links",
-    header: header,
-    mobile_header: header,
-    enabled: true,
-    user_id: User.first.id,
-    head_tag: head_tag
-  })
-  # cleanup old customizations
-  SiteCustomization.where.not(id: sitepoint_site_customization.id).delete_all
+  unless User.exists?
+    sitepoint_site_customization = SiteCustomization.find_or_create_by({
+      name: "SitePoint Crawler links",
+      header: header,
+      mobile_header: header,
+      enabled: true,
+      user_id: User.first.id,
+      head_tag: head_tag
+    })
+    # cleanup old customizations
+    SiteCustomization.where.not(id: sitepoint_site_customization.id).delete_all
```

Comment:
We might want to change this to only look for customisations with the same `name` as we are injecting, so that we don't clear out any ones that are added by other things

```ruby
SiteCustomization.where(name: sitepoint_site_customization.name).
  where.not(id: sitepoint_site_customization.id).
  delete_all
```

or something similar. Thoughts @tancnle ?